docs/pages/why-web-components.md

```diff
@@ -0,0 +1,124 @@
+
+Components help developers and designers save time. They help you avoid rewriting redundant code. You can also easily plug them into and remove them from code using HTML tags.
+
+The California Design System uses the minimal amount of tech required for each component. This is&nbsp;often just HTML &amp; CSS. When we need JavaScript, we take advantage of the component model built into the browser. We use the custom elements part of the web components specification. This allows us to define new HTML tags like &lt;cagov-accordion&gt; or &lt;cagov-page-feedback&gt;, which are inert until upgraded by provided JavaScript.
+
+### Examples of our components
+
+The homepage also uses the <a href="https://designsystem.webstandards.ca.gov/components/feature-card/readme/">feature card</a>&nbsp;component. This component requires only HTML & CSS. 
+
+<img alt="" src="/content/img/screenshot-feature.jpg">
+
+In some cases, we can embed components in other components. For example, we put a button in the feature card.
+
+We use the <a href="/components/page-feedback/readme/">page feedback component</a> at the bottom of every page. This component requires HTML, CSS and JavaScript. It uses a custom element tag associated with a JavaScript class.
+
+<img alt="" src="/content/img/screenshot-feedback.jpg">
+
+## Web component advantages
+
+### Web components work anywhere
+
+<a href="https://custom-elements-everywhere.com/">All framework support web components</a>. You get this component model with its API built into all browsers. If you still need to support Internet Explorer, a polyfill is available. We use @webcomponents/webcomponentsjs on covid19.ca.gov.
+
+### Web components help ensure longevity of code
+
+Preferred web stacks evolve. Many mature organizations maintain applications written in a variety of frameworks. Each rewrite or launch is an opportunity to update codebases to the &ldquo;modern approach,&rdquo; but that changes over time.
+
+Web components allow us to take our frontend widgets&nbsp;off this cycle&nbsp;of getting rebuilt in the newest framework flavor. Standardized features are built into all browsers. This means they&rsquo;re supported forever. We are not subject to breaking changes in new versions of frameworks. We also do not worry about developers who prefer to write widgets that only work with a specific underlying technology.
+
+### Easier component management
+
+Publishing each component individually lets us take advantage of separate versioning. Site owners get <a href="https://github.com/dependabot">dependabot</a>&nbsp;notifications when there are updates to each npm package. This comes with associated <a href="https://semver.org/">semver</a> compliant version numbers and published changelogs. Site owners can decide if they want to upgrade or ignore the change on their own timeline.
+
+Owners can add, upgrade, or remove components 
+
```

Comment:
Remove line break.

Reply:
updated

Reply:
One more tweak - "Site owners can get dependabot" instead of "Site owners get dependabot". Since it doesn't happen automagically.

Reply:
updated